subprojects/build-init/build-init.gradle.kts

```diff
@@ -59,7 +59,7 @@ tasks {
 
             findLatest("scalatest", "org.scalatest:scalatest_${versionProperties["scala"]}:(3.0,)", versionProperties)
             findLatest("scala-xml", "org.scala-lang.modules:scala-xml_${versionProperties["scala"]}:latest.release", versionProperties)
-            findLatest("groovy", "org.codehaus.groovy:groovy:(2.4,2.5]", versionProperties)
+            findLatest("groovy", "org.codehaus.groovy:groovy:(2.5,2.6]", versionProperties)
```

Comment:
I don't quite understand the version ranges. Doesn't the range mean that `2.6` would be included and `2.5` excluded? I think it should be the other way round, right?

Reply:
Right. Seems they were wrong? before.
It's good that got documented recently :) https://github.com/gradle/gradle/commit/44630395e54138108e98cd69ccc3a355b0817a24#diff-60edd0ca069e1e63e58f23bc7df3878bR35